user-guide/Advanced_Functionality/Databases/Supported_System_Data_Storage_Architectures/Migrating_the_general_database_to_a_DMS_Cassandra_cluster.md

```diff
@@ -61,6 +61,106 @@ During the migration, each DMA will go through the following stages:
 
 ### Running the migration
 
+#### [Running a migration with bespoke Elasticsearch data](#tab/tabid-1)
+
+In case your system contains bespoke Elasticsearch data or SRM data, use the procedure below.
+
+1. Follow the step-by-step guide on [taking and restoring snapshots](xref:Configuring_Elasticsearch_backups_Windows_Linux).
+
+   Make sure the snapshot has been [restored](xref:Configuring_Elasticsearch_backups_Windows_Linux#taking-the-snapshot) on the target Elasticsearch cluster.
+
+1. Enter `http://[IP address]:9200/_cat/indices` in your browser's address bar. Replace "[IP address]" with your IP address.
+
+   Take note of the prefixes used in the indices. You will need this information later. The default prefix is "dms".
+
+   ![Prefix snapshot](~/user-guide/images/Prefix_Snapshot.png)
+
+1. Stop your DataMiner System.
+
+1. Open the *C:\Skyline DataMiner\db.xml* file and check if `<DataBase type="Elasticsearch">` exists.
+
+   Example:
+
+   ```xml
+   <DataBases xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.skyline.be/config/db">
+      ...
+      <DataBase active="true" search="true" type="Elasticsearch">
+         <DBServer>10.2.1.5</DBServer>
+         <UID/>
+         <PWD/>
+      </DataBase>
+      ...
+   </DataBases>
+   ```
+
+   If you find `<DataBase type="Elasticsearch">`, replace the entries in the `<DBServer>` tag with a list of all the nodes in the target Elasticsearch cluster, separated by commas.
```

Comment:
What should you do if the `<DataBase type="Elasticsearch">`  entry does not exist?

Reply:
If there's Custom or SRM data, these are stored in the Elasticsearch database. 
If this tag is not here, we assume a default Cassandra to Cassandra Cluster migration procedure (i.e. do not take a snapshot, move all data using the SLCCMigrator tool).